packages/openapi-generator/src/generator.ts

```diff
@@ -30,30 +32,34 @@ const logger = createLogger('openapi-generator');
  * Generates files using the OpenApi Generator CLI and wraps the resulting API in an SDK compatible API.
  * @param options Options to configure generation.
  */
-export async function generate(options: GeneratorOptions): Promise<void> {
+export async function generate(options: GeneratorOptions): Promise<void[]> {
```

Comment:
Does that make sense? What does is mean?

Reply:
This is a result of the Promise.all(promises from the N spech files), which results in a single promise with an array giving the results. I added now a `.then(()=>undefined)` to get a single promise out of it.